.storybook/manager.ts

```diff
@@ -3,7 +3,7 @@ import {create} from '@storybook/theming';
 
 const theme = create({
   base: 'light',
-  brandTitle: 'Sentry Styleguide',
+  brandTitle: 'Sentry UI System',
```

Comment:
"UI system" seems appropriate here because we're referring to a group of UI assets/components that are related to one another though a common design language (hence "system"). This may change if we find a more appropriate name.